plugins/time-resources/package.json

```diff
@@ -63,6 +63,7 @@
     "@hcengineering/time": "^0.6.0",
     "fast-equals": "^5.0.1",
     "@hcengineering/activity": "^0.6.0",
-    "@hcengineering/activity-resources": "^0.6.1"
+    "@hcengineering/activity-resources": "^0.6.1",
+    "@hcengineering/workbench-resources": "^0.6.1"
```

Comment:
This is not super good, we probable need to move all navigator basic components into view-resources.